docs/products/kafka/howto/enable-kafka-tiered-storage.md

```diff
@@ -94,42 +73,82 @@ Kafka service using the [Aiven CLI](/docs/tools/cli):
    team](mailto:sales@aiven.io). This feature is an early availability and requires
```

Comment:
"is activated for your account" -> "is activated for your organization"?

Also, if this is GA, this sentence needs to be removed I guess "This feature is an early availability ...."